plugins/wazuh_check_updates/public/components/updatesNotification.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  EuiBadge,
+  EuiBottomBar,
+  EuiButton,
+  EuiCheckbox,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHeaderLink,
+  EuiText,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { useAvailableUpdates, useUserPreferences } from '../hooks';
+import { getCurrentAvailableUpdate } from '../utils';
+
+export const UpdatesNotification = () => {
+  const [isDismissed, setIsDismissed] = useState(false);
+  const [dismissFutureUpdates, setDismissFutureUpdates] = useState(false);
+
+  const {
+    userPreferences,
+    error: userPreferencesError,
+    isLoading: isLoadingUserPreferences,
+    updateUserPreferences,
+  } = useUserPreferences();
+
+  const {
+    availableUpdates,
+    error: getAvailableUpdatesError,
+    isLoading: isLoadingAvailableUpdates,
+  } = useAvailableUpdates();
+
+  if (userPreferencesError) {
+    console.log(userPreferencesError);
```

Comment:
**issue**: This could be done in each render of the component.

Reply:
console.log deleted